### ISSUE TITLE
FbxImporter fails on Ubuntu 18

### ISSUE THREAD

**Issue by ChrisTeister** [MonoGame/MonoGame#7339]
I get the following error, when I try to import FBX models:

> /home/chris/RiderProjects/Shooter/Desktop/content/models/Ship.fbx
/home/chris/RiderProjects/Shooter/Desktop/content/models/Ship.fbx: error: Importer 'FbxImporter' had unexpected failure!
Assimp.AssimpException: Error loading unmanaged library from path: /home/chris/.dotnet/tools/.store/dotnet-mgcb-editor/3.8.0.1641/dotnet-mgcb-editor/3.8.0.1641/tools/netcoreapp3.1/any/runtimes/linux-x64/native/libassimp.so
libminizip.so.1: cannot open shared object file: No such file or directory
   at Assimp.Unmanaged.UnmanagedLibrary.UnmanagedLinuxLibraryImplementation.NativeLoadLibrary(String path)
   at Assimp.Unmanaged.UnmanagedLibrary.UnmanagedLibraryImplementation.LoadLibrary(String path)
   at Assimp.Unmanaged.UnmanagedLibrary.LoadLibrary(String libPath)
   at Assimp.Unmanaged.UnmanagedLibrary.LoadLibrary()
   at Assimp.Unmanaged.UnmanagedLibrary.LoadIfNotLoaded()
   at Assimp.Unmanaged.AssimpLibrary.CreatePropertyStore()
   at Assimp.AssimpContext.CreateConfigs()
   at Assimp.AssimpContext.PrepareImport()
   at Assimp.AssimpContext.ImportFile(String file, PostProcessSteps postProcessFlags)
   at Microsoft.Xna.Framework.Content.Pipeline.OpenAssetImporter.Import(String filename, ContentImporterContext context) in C:\BuildAgents\MonoGameWin1\work\f7381a85a626990\MonoGame.Framework.Content.Pipeline\OpenAssetImporter.cs:line 291
   at Microsoft.Xna.Framework.Content.Pipeline.FbxImporter.Import(String filename, ContentImporterContext context) in C:\BuildAgents\MonoGameWin1\work\f7381a85a626990\MonoGame.Framework.Content.Pipeline\FbxImporter.cs:line 24
   at Microsoft.Xna.Framework.Content.Pipeline.ContentImporter`1.Microsoft.Xna.Framework.Content.Pipeline.IContentImporter.Import(String filename, ContentImporterContext context) in C:\BuildAgents\MonoGameWin1\work\f7381a85a626990\MonoGame.Framework.Content.Pipeline\ContentImporter.cs:line 45
   at MonoGame.Framework.Content.Pipeline.Builder.PipelineManager.ProcessContent(PipelineBuildEvent pipelineEvent) in C:\BuildAgents\MonoGameWin1\work\f7381a85a626990\MonoGame.Framework.Content.Pipeline\Builder\PipelineManager.cs:line 669

Steps to reproduce:
1. Download and unpack http://rbwhitaker.wdfiles.com/local--files/model-library/SimpleShip.zip
2. Launch the mgcb-editor by doube clicking on a Content.mgcb file
3. Add the unpacked fbx file
4. Hit the "Build" button

#### What version of MonoGame does the bug occur on:
- MonoGame 3.8.0.16.41

#### What operating system are you using:
- Ubuntu 18.04.5 LTS

#### What MonoGame platform are you using:
- DesktopGL

**Comment by rejurime** [comment]
Hi!
If you try installing ```sudo apt install libassimp-dev```

**Comment by ChrisTeister** [comment]
Thanks a lot! That fixed the first error. Did I miss that somewhere in the documentation?

Unfortunately I get this error now:
/home/chris/RiderProjects/MonoGameTest/MonoGameTest/Content/Ship/Ship.fbx
	/home/chris/RiderProjects/MonoGameTest/MonoGameTest/Content/Ship/*0
/home/chris/RiderProjects/MonoGameTest/MonoGameTest/Content/Ship/Ship.fbx: error: The source file '/home/chris/RiderProjects/MonoGameTest/MonoGameTest/Content/Ship/*0' does not exist!

**Comment by rejurime** [comment]
This dependency is not in the documentation, I think it is implicit somewhere, but I had already come across that and with that I could solve it. This model is from a tutorial that I tried to make at the time (in ubuntu and windows), the model is binary, so I cannot see internally how it is composed. I used a tool (Autodesk FBX Converter) to convert old fbx or binaries to acsii format so that I could explore it inside and it ended up fixing it.

In Ubuntu 20.04 I had problems importing models with textures: https://community.monogame.net/t/problem-loading-models-with-textures-in-ubuntu-20-04-with-mg-3-8/
